docs/docs/aztec/concepts/fees.md

```diff
@@ -4,6 +4,68 @@ sidebar_position: 0
 tags: [fees]
 ---
 
-This section is currently being updated and will supplemented by information in the corresponding section of the protocol specifications.
+import { Why_Fees } from '/components/snippets';
 
-For a guide on how to pay fees programmatically, see [Paying fees in TypeScript](../../developers/guides/js_apps/pay_fees).
+<Why_Fees />
+
+In a nutshell, the pricing of transactions transparently accounts for:
+- L1 costs, including L1 execution of a block, and data availability via blobs,
+- L2 node operating costs, including proving
+
+This is done via multiple variables and calculations explained in detail in the protocol specifications.
+
+## Terminology and factors
+
+Familiar terms from Ethereum mainnet as referred to on the Aztec network:
+
+| Mainnet     | Aztec              | Description |
```

Comment:
can we replace "Mainnet" with "Ethereum"
Seeing mainnet is ambiguous. can also mean Aztec mainnet especially since you only mention "Aztec" later